client/state/data-layer/wpcom/activity-log/rewind/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { pick } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	REWIND_STATUS_REQUEST,
+} from 'state/action-types';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+const fetchRewindStatus = ( { dispatch }, action ) => {
+	dispatch( http( {
+		method: 'GET',
+		path: `/activity-log/${ action.siteId }/rewind`,
+	}, action ) );
+};
+
+const fromApi = response => ( {
+	active: response.use_rewind,
+	firstBackup: response.first_backup_when,
+	error: { message: response.error },
+} );
+
+const updateRewindStatus = ( { dispatch }, { siteId }, next, data ) => {
+	dispatch( {
+		type: REWIND_STATUS_REQUEST,
```

Comment:
Ok, then I've been misunderstanding. I was thinking it was now possible to use one action type for the whole cycle.  This [diagram](https://github.com/Automattic/wp-calypso/blob/master/client/state/data-layer/wpcom-http/README.md) has `LIKE_POST` being fired both to trigger the http request and on a success response.